cypress/support/commands.ts

```diff
@@ -128,8 +127,15 @@ Cypress.Commands.add("saveToDatabase", () => {
 });
 
 Cypress.Commands.add("enterEditMode", () => {
+  cy.get("[data-cy=revision-select]").select("wersja aktualna").should("have.value", "actual");
   cy.get("[data-cy=edit-mode-button]").click();
-  return cy.get("[data-cy=nurseShiftsTable]", { timeout: 10000 });
+  cy.get("[data-cy=nurseShiftsTable]", { timeout: 10000 });
+  cy.getWorkerShift({
```

Comment:
To jest ten hack o którym mówiłeś?
Że nie da się przy pierwszym wejściu zmienić zmianę

Reply:
Tak, dokładnie. Nie podoba mi się on bardzo i wolałbym się tego pozbyć. Mam nadzieję, że uda Ci się poprawić testy z developa i nie będzie takich problemów i będę mógł to wywalić.

Reply:
Teraz to sprawdzę

Reply:
Wygląda na to, że problem był z tym, że select nadal miał focus i przez to kliknięcie poza selectem robił jedynie to, że usuwało focus z niego, zamiast wykonywać akcję an jakimś elemencie.
I przez to hack z dodatkowym kliknięciem pomagał. 

Jedyne czego nie rozumiem, to dlaczego testy normalnie przechodziły lokalnie.

![image](https://user-images.githubusercontent.com/21079319/107882303-cd412a00-6ee8-11eb-98a7-990f8442b6ee.png)

Reply:
Nie wiem dlaczego, ale u nas to nie działa